addons/godot-xr-tools/functions/function_teleport.gd

```diff
@@ -4,9 +4,6 @@ extends KinematicBody
 
```

Comment:
Same with teleport? should we add a class name?

Reply:
Modified to class_name XRToolsFunctionTeleport with icon.